javalib/src/main/scala/java/util/LinkedList.scala

```diff
@@ -16,17 +16,17 @@ class LinkedList[E]() extends AbstractSequentialList[E]
   private var head: Node[E] = null
   private var last: Node[E] = null
 
-  private var _size = 0
+  private var _size: Double = 0
```

Comment:
It would be good to have a (non-ScalaDoc) comment above this declaration of `_size` explaining why it is a `Double`.